documentation/jetty-documentation/src/main/asciidoc/operations-guide/xml/xml-syntax.adoc

```diff
@@ -187,12 +189,38 @@ This is equivalent to the following Java code:
 var myhost = InetAddress.getByName("jdk.java.net");
 ----
 
+The `class` attribute can also be used to specify the Java class or interface to use to lookup the non-``static`` method name.
```

Comment:
```
The `class` argument ...
```
(it's not always an attribute)

Reply:
Don't like "argument", added  "(or `<Class>` element)" instead.